app/models/api_docs/service.rb

```diff
@@ -27,6 +27,7 @@ class ApiDocs::Service < ApplicationRecord
   scope :published, -> { where(published: true) }
   scope :accessible, -> { joining { service.outer }.where.has { (service_id == nil) | (service.state != ::Service::DELETE_STATE) } }
   scope :without_service, -> { where(service_id: nil) }
+  scope :permitted_for, ->(user) { where.has { (service_id == nil) | service_id.in(user.accessible_services.select(:id)) } }
```

Comment:
The code is good but this scope probably needs a unit test as well 🙈 Plus, let's run the tests in the 3 DBs, just in case 🙏

Reply:
Done